<?xml version = "1.0"?>
<launch>
    <node pkg = "chibi22_a" name = "local_map_creator" type = "local_map_creator" output="screen">
        <param name = "hz" type = "int" value = "10"/>
        <param name = "map_size" type = "double" value = "4"/>
        <param name = "laser_density" type = "double" value = "1.5"/>
        <param name = "ignore_angle_mergin" type = "double" value = "0.2"/>
        <param name = "map_reso" type = "double" value = "0.05"/>
        <param name = "roomba_radius" type = "double" value = "0.2"/>
    </node>

    <node pkg="tf2_ros" type="static_transform_publisher" name="static_transform_publisher" args="0 0 0 0 0 0 1 /base_link /laser"/>
    <arg name="map_file" default="$(find chibi22_a)/map/map.yaml"/>
    <node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)" output="screen"/>
    <node pkg="rviz" name="rviz" type="rviz" args="-d$(find chibi22_a)/config/rviz/local_map_creator.rviz" output="screen"/>
    <node pkg="rosbag" name="play" type="play" args="--clock -r 3 /home/yoshiki/bagfiles/test.bag"/>
    <rosparam param="/use_sim_time">true</rosparam>
</launch>
